components/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/add_service_provider.jsp

```diff
@@ -1176,8 +1179,7 @@
                                     <td colspan="2" title="Select Enable Response Signing to sign the SAML2 Responses returned after the authentication process">
                                         <input type="checkbox" name="enableResponseSignature" value="true"
                                                onclick="disableResponseSignature(this);"
-                                                <%=(isEditSP && provider.getDoSignResponse() ? "checked=\"checked\"" : "")%> />
-                                            <%--<input type="hidden" name="enableResponseSignature" value="true"/>--%>
+                                                <%=(isResponseSigningEnabled(isEditSP, provider) ? "checked" : "")%> />
```

Comment:
shall we replace inline if with an if else clause?

Reply:
Any reason  for your suggestion ? IMO  if possible   I would replace others places also with inline if because  IMO inline if is much readable  in this kind of situations WDYT ?